azurerm/internal/services/portal/parse/tenant_configuration.go

```diff
@@ -0,0 +1,51 @@
+package parse
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type TenantConfigurationId struct {
```

Comment:
we should update this file and these names?
```suggestion
type PortalTenantConfigurationId struct {
```

Reply:
Updated